mindsdb/api/executor/command_executor.py

```diff
@@ -1300,17 +1303,35 @@ def answer_drop_tables(self, statement, database_name):
 
         return ExecuteAnswer()
 
-    def answer_create_view(self, statement, database_name):
+    def answer_create_or_alter_view(self, statement: ASTNode, database_name: str) -> ExecuteAnswer:
+        """Process CREATE and ALTER VIEW commands
+
+        Args:
+            statement (ASTNode): data for creating or altering view
+            database_name (str): name of the current database
+
+        Returns:
+            ExecuteAnswer: answer for the command
+        """
         project_name = database_name
-        # TEMP
-        if isinstance(statement.name, Identifier):
+
+        if isinstance(statement.name, str):
```

Comment:
I think 'name' can't be a string here

Reply:
found, it is done here
https://github.com/mindsdb/mindsdb_sql_parser/blob/7f134d2facf1f795e65d8cbfaa66e41b8143ab31/mindsdb_sql_parser/ast/mindsdb/create_view.py#L15